### DISCUSSION TITLE
开发者例会议题收集（2023-03-21）

### DISCUSSION THREAD

**Discussion by chuntaojun** [polarismesh/polaris#1011] [Meeting]
# 开发者例会议题收集（2023-03-21）

## 会议信息

andrewshan(单家骏) 邀请您参加腾讯会议
会议主题：PolarisMesh社区双周例会
会议时间：2023/02/21 20:00-20:30 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/12/12-2023/03/21 20:00-20:30, 每两周 (周二)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/bi26r1VIAx2y

#腾讯会议：825-9154-7185

### 分享

- 上次周会相关任务进度情况：https://github.com/polarismesh/polaris/discussions/915
- 社区 commiter seansyyu 分享 spring cloud tencent 对接北极星接口级熔断以及服务级熔断
- 社区 commiter alexwanglei 分享北极星配置加密的技术方案
- 北极星 1.15 版回顾 & 北极星 1.16 版本功能特性预告

#### 服务治理

- https://github.com/polarismesh/polaris/issues/994
- https://github.com/polarismesh/polaris/issues/993

#### 使用问题

- https://github.com/polarismesh/polaris/issues/944

#### 使用体验

- https://github.com/polarismesh/polaris/issues/984

**Comment by taolive** [comment]
配置中心的导入导出很实用的一个功能

**Comment by lei32323** [comment]
envoy接入注册中心, xds下发配置支持单个domain下多path访问
